torch/csrc/autograd/variable.cpp

```diff
@@ -368,4 +373,56 @@ unsigned VariableHooks::_register_hook(const Tensor& self, std::function<Tensor(
   return idx;
 }
 
+TORCH_API void handle_view_on_rebase(DifferentiableViewMeta* diff_view_meta, bool indirect) {
```

Comment:
what does this function do? What is `indirect`?

Reply:
This is described in the header: "indirect=true means that the caller is not doing the inplace, but the inplace happened somewhere else."
Should I move the description in the cpp file?

Reply:
Don't declare `TORCH_API` in the `cpp` file, it's redundant with the definition from the header.

Docs in the header is OK but this seems like a very implementation-detail specific function that I wouldn't expect as part of the public API.

Reply:
We need this to be available in `VariableTypeUtils`. That is why I added the `TORCH_API`. But maybe this is not needed? I will try and remove if not needed.